init/bash

```diff
@@ -29,6 +29,11 @@ if [ $? -eq 0 ]; then
     show_msg "Prepending site path $EESSI_SITE_MODULEPATH to \$MODULEPATH..."
     module use $EESSI_SITE_MODULEPATH
 
+    if [ ! -z ${EESSI_MODULEPATH_ACCEL} ]; then
```

Comment:
`init/modules/EESSI/2023.06` is also going to need to support this. I can make a PR tomorrow.

Reply:
Of course, but let's tackle that in a follow-up PR?

Reply:
I will try to get to that today then